server.js

```diff
@@ -44,12 +45,8 @@ var renderHtml = function(url, cb) {
     }; //For service purposes only
 
     page.onInitialized = function() {
-       page.evaluate(function() {
-            setTimeout(function() {
-                console.debug("setTimeout() rang, calling callPhantom()");
-                window.callPhantom();
-            }, 10000); //If no callPhantom script was called on the page, then wait 10secs and load page anyway.
-        });
+       watch.angular(page); //Remove that if you're not using angular
```

Comment:
Sure. That makes sense. We're not building an any-and-all-should-work thing here. We _know_ that _we_ are using Angular,.. so this is a good solution.